agent/util-scripts/pbench-register-tool-set

```diff
@@ -161,7 +161,7 @@ for tool_name in ${the_tool_set}; do
 done
 # low overhead perf
 if [[ ${reg_perf} -ne 0 ]]; then
-	pbench-register-tool --name=perf${group_arg}${install_arg}${remotes_arg}${labels_arg} -- --record-opts="record -a --freq=100"
+	pbench-register-tool --name=perf${group_arg}${install_arg}${remotes_arg}${labels_arg} -- --record-opts="'record -a --freq=100'"
```

Comment:
Double quotes here appear to be redundant.

Reply:
Looks are deceiving.  Try taking the double quotes out to see if the unit tests pass?  When you do, because `bash` processes the single quotes, the stored string no longer has any quotes, and so it can't be passed correctly to the tool when it is invoked.

Reply:
Someone should write an article (or perhaps a book :-) ) on quoting problems in bash: "The Joy of Quoting" - guaranteed best-seller...

Reply:
If you happen to have a copy of Kernighan & Pike's "Unix Programming Environment", read p.128. There is a PDF online at https://scis.uohyd.ac.in/~apcs/itw/UNIXProgrammingEnvironment.pdf - but you have to go to p.140 of the PDF to find p.128 of the book.